recipe-server/client/control_new/tests/components/recipes/test_SwitchBox.js

```diff
@@ -10,9 +11,46 @@ describe('<SwitchBox>', () => {
     value: null,
   };
 
+  const factory = (customProps = {}) => mount(<SwitchBox {...props} {...customProps} />);
+
   it('should work', () => {
-    const wrapper = () => shallow(<SwitchBox {...props} />);
+    expect(factory).not.toThrow();
+  });
+
+  describe('the label', () => {
+    it('should not render if there is no label', () => {
+      const wrapper = factory();
+      expect(wrapper.find('.label').length).toBe(0);
+    });
+
+    it('should render the passed in children', () => {
+      const label = 'Root beer\'s flavor comes from sassafras and/or sarsaparilla!';
```

Comment:
Nit: I'm pretty sure that in  this case the linter will let you use double quotes to avoid the `\'` escape. This doesn't need to block this PR though.